### PR TITLE
Registration and user profile documentation reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: docs/user/profile.rst [hunk 1]
```diff
@@ -4,36 +4,40 @@ Registration and user profile
...
 
-By default, this will show the components of projects you are watching,
-cross-referenced with your preferred languages.  You can switch to different
-views using the navigation tabs.
+Components of projects you are watching are shown by default, and
+cross-referenced with your preferred languages.
+
+.. hint::
+
+You can switch to different views using the navigation tabs.
```
Comment (nijel) [inline docs/user/profile.rst]:
```suggestion
    You can switch to different views using the navigation tabs.
```

The `.. hint::` content has to be indented, see https://docutils.sourceforge.io/docs/ref/rst/directives.html#hint

Comment (orangesunny) [reply docs/user/profile.rst]: @comradekingu can you add what you wanted there?

File: docs/user/profile.rst [hunk 2]
```diff
@@ -101,61 +110,63 @@ list`, you have to select which component list will be displayed from the
...
-By default Weblate does display source code in the web browser configured in the
-:ref:`component`. By setting :guilabel:`Editor link` you can override this to
-use your local editor to open the source code where translated strings are being
-used. You can use :ref:`markup`.
+A source code link is shown in the web-browser configured in the
+:ref:`component` by default.
+
+.. hint::
+
+By setting the :guilabel:`Editor link`, you use your local editor to open the VCS source code
```
Comment (comradekingu) [inline docs/user/profile.rst]:
```suggestion
    By setting the :guilabel:`Editor link`, you use your local editor to open the VCS source code
```

File: docs/user/profile.rst [hunk 3]
```diff
@@ -4,36 +4,40 @@ Registration and user profile
...
 .. image:: /images/dashboard-dropdown.png
 
-The menu will show several options:
+The menu has these options:
 
-- :guilabel:`Projects` > :guilabel:`Browse all projects` in the main menu (or menu :guilabel:`Tools` > :guilabel:`All projects` in the Dashboard) will show translation status of all projects on the
-  Weblate instance.
+- :guilabel:`Projects` > :guilabel:`Browse all projects` in the main menu (or in the menu
+:guilabel:`Tools` → :guilabel:`All projects` in the Dashboard) showing translation status
+for each project on the Weblate instance.
```
Comment (nijel) [inline docs/user/profile.rst]:
```suggestion
  :guilabel:`Tools` → :guilabel:`All projects` in the Dashboard) showing translation status
  for each project on the Weblate instance.
```

File: docs/user/profile.rst [hunk 4]
```diff
@@ -101,61 +110,63 @@ list`, you have to select which component list will be displayed from the
...
+A source code link is shown in the web-browser configured in the
+:ref:`component` by default.
+
+.. hint::
+
+    By setting the :guilabel:`Editor link`, you use your local editor to open the VCS source code
+file of translated strings. You can use :ref:`markup`.
 
 Usually something like ``editor://open/?file={{filename}}&line={{line}}`` is a good
 option.
```
Comment (nijel) [inline docs/user/profile.rst]:
```suggestion
    By setting the :guilabel:`Editor link`, you use your local editor to open the VCS source code
file of translated strings. You can use :ref:`markup`.

    Usually something like ``editor://open/?file={{filename}}&line={{line}}`` is a good option.
```

File: docs/user/profile.rst [hunk 5]
```diff
@@ -101,61 +110,62 @@ list`, you have to select which component list will be displayed from the
...
-:ref:`component`. By setting :guilabel:`Editor link` you can override this to
-use your local editor to open the source code where translated strings are being
-used. You can use :ref:`markup`.
+A source code link is shown in the web-browser configured in the
+:ref:`component` by default.
+
+.. hint::
+
+    By setting the :guilabel:`Editor link`, you use your local editor to open the VCS source code
+file of translated strings. You can use :ref:`markup`.
```
Comment (nijel) [inline docs/user/profile.rst]:
```suggestion
    file of translated strings. You can use :ref:`markup`.
```

File: docs/user/profile.rst [hunk 6]
```diff
@@ -43,35 +47,40 @@ In addition, the drop-down can also show any number of *component lists*, sets
...
+The user profile contains your preferences. Name and e-mail address is used in VCS commits, so keep this info accurate.
 
 
 .. note::
 
-    All language selections offer only languages which are currently being
-    translated. If you want to translate to another language, please request it
-    first on the project you want to translate.
+    All language selections only offer currently translated languages.
+
```
Comment (nijel) [inline docs/user/profile.rst]:
```suggestion
    All language selections only offer currently translated languages.

```

**Pull request merged**